bundle/config/mutator/apply_presets.go

```diff
@@ -221,6 +222,19 @@ func (m *applyPresets) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnos
 		dashboard.DisplayName = prefix + dashboard.DisplayName
 	}
 
+	if config.IsExplicitlyEnabled((b.Config.Presets.InPlaceDeployment)) {
```

Comment:
I'd also recommend creating a separate mutator for this functionality instead of adding it into `apply_presets`

Reply:
Yeah that makes sense. I'll leave it up to your discretion since in-place deployments are just for mode development.  A separate mutator can be nice if this logic for in-place deployments really becomes complex as we handle more edge cases.